format/apple/bplist/ns_keyed_archiver.jq

```diff
@@ -0,0 +1,70 @@
+def from_ns_keyed_archiver:
+  (  . as {
+      "$objects": $objects,
+      "$top": {root: $root}
+      #"$top": {"796BFF22-6712-4486-A32C-A1C5DB3273BA": $root}
```

Comment:
Hmm guess need to clean this up somehow

Reply:
Overloaded `from_ns_keyed_archiver` so that the user can specify an optional root value from which to begin decoding